File: Web.Admin/src/api/services/merchant/models/QueryMerchantPagedOutput.ts
import { PaymentChannelEnum } from "@/api/enums/PaymentChannelEnum";

/**
 * Fast.Center.Service.Merchant.Dto.QueryMerchantPagedOutput 获取商户号分页列表输出
 */
export interface QueryMerchantPagedOutput {
  /**
   * 商户号Id
   */
  merchantId?: number;
  /**
   * 
   */
  merchantType?: PaymentChannelEnum;
  /**
   * 商户号
   */
  merchantNo?: string;
  /**
   * 备注
   */
  remark?: string;
  /**
   * 创建者用户名称
   */
  createdUserName?: string;
  /**
   * 创建时间
   */
  createdTime?: Date;
  /**
   * 更新者用户名称
   */
  updatedUserName?: string;
  /**
   * 更新时间
   */
  updatedTime?: Date;
}

File: Api.Server/src/Services/Center/Merchant/Dto/QueryMerchantPagedOutput.cs
namespace Fast.Center.Service.Merchant.Dto;

/// <summary>
/// <see cref="QueryMerchantPagedOutput"/> 获取商户号分页列表输出
/// </summary>
public class QueryMerchantPagedOutput : PagedOutput
{
    /// <summary>
    /// 商户号Id
    /// </summary>
    public long MerchantId { get; set; }

    /// <summary>
    /// 商户号类型
    /// </summary>
    public PaymentChannelEnum MerchantType { get; set; }

    /// <summary>
    /// 商户号
    /// </summary>
    public string MerchantNo { get; set; }

    /// <summary>
    /// 备注
    /// </summary>
    public string Remark { get; set; }
}